docs/design/README.md

```diff
@@ -108,6 +108,12 @@ The sequence diagram for a Digital Twin Consumer using the Invoke operation.
 
 ![Sequence Diagram](diagrams/invoke_sequence.svg)
 
+#### Stream
+
+The sequence diagram for a Digital Twin Consumer using the Stream operation.
```

Comment:
I'd suggest ending with a colon instead of a period or adding "is shown below"

Reply:
Added "is shown below".